client/js/util.js

```diff
@@ -1,4 +1,7 @@
 // @flow
+
+import fastdom from '../js/utils/fastdom-promise';
```

Comment:
Any reason to be using `fastdom-promise` vs `fastdom` <- I think we use the latter elsewhere, so adding this will add extras bits to our client JS.